addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -610,10 +610,14 @@ $line-height-truncate: 1.25em;
     }
 }
 
-// Enforce the height of the lesson header, so the DOM element size do not change
-// when the completion progressbar become visible / hidden
-.o_wslides_lesson_header_container {
-    height: 80px;
+@media (max-width: 576px) {
+    .hidden-progressbar-completed-tag {
+        display: none;
+    }
+}
```

Comment:
You should be able to nest this inside the tag below for readability and compactness.
Also, you may use the bootstrap breakpoints instead of "hardcoding" `576px`. See https://getbootstrap.com/docs/5.0/layout/breakpoints/